Makefile

```diff
@@ -8,10 +8,10 @@ PULL_PARAM=$(shell if [ "${CONTAINER_COMMAND}" = "podman" ];then echo "--pull-al
 
 SKIPPER_PARAMS ?= -i
 
-# bm-inventory
-BMI_BRANCH := $(or $(BMI_BRANCH), "master")
-BMI_REPO := $(or $(BMI_REPO), "https://github.com/filanov/bm-inventory")
-SERVICE := $(or $(SERVICE), quay.io/ocpmetal/bm-inventory:latest)
+# assisted-service
+SERVICE_BRANCH := $(or $(SERVICE_BRANCH), "master")
```

Comment:
Maybe it should be `ASSISTED_SERVICE_BRANCH` and the same for the repo? it will be clearer?